src/java.base/share/classes/jdk/internal/foreign/abi/ppc64/aix/AixPPC64Linker.java

```diff
@@ -64,8 +68,8 @@ protected MethodHandle arrangeDowncall(MethodType inferredMethodType, FunctionDe
     }
 
     @Override
-    protected MemorySegment arrangeUpcall(MethodHandle target, MethodType targetType, FunctionDescriptor function, SegmentScope scope) {
-        return CallArranger.arrangeUpcall(target, targetType, function, scope);
+    public UpcallStubFactory arrangeUpcall(MethodType mt, FunctionDescriptor cDesc) {
```

Comment:
This method should continue to be `protected`; likewise implementations in `SysVPPC64leLinker` and `SysVS390xLinker`.

Reply:
Restored